packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -587,7 +589,7 @@ export class VMixDevice extends DeviceWithState<VMixStateExtended, DeviceOptions
 		return this._doOnTime.getQueue()
 	}
 
-	private _addToQueue(commandsToAchieveState: Array<VMixStateCommandWithContext>, time: number) {
+	public addToQueue(commandsToAchieveState: Array<VMixStateCommandWithContext>, time: number) {
```

Comment:
We shouldn't carelessly add things to the public API of these devices. If using callbacks this isn't necessary.